src/module/capturer/local_video.cpp

```
@@ -85,6 +37,73 @@ struct LocalVideo::Impl {
         }
     };
 
+    auto step_frame_locked(int offset) -> void {
+        if (!capturer.has_value()) return;
```

Cppcheck (2.21.0)

[error] Found an exit path from function with non-void return type that has missing return statement

(missingReturn)